pkg/ingester/wal.go

```diff
@@ -240,7 +298,7 @@ func (w *walWrapper) performCheckpoint(immediate bool) (err error) {
 	if err := os.MkdirAll(checkpointDirTemp, 0777); err != nil {
 		return errors.Wrap(err, "create checkpoint dir")
 	}
-	checkpoint, err := wal.New(nil, nil, checkpointDirTemp, true)
+	checkpoint, err := wal.New(nil, nil, checkpointDirTemp, false)
```

Comment:
No need for compression anymore?

Reply:
We could, but that causes additional CPU usage. We can turn it on later if very necessary (TSDB supports co-existence of compressed and uncompressed records, so it is fine to move around).

Reply:
Is this worth mentioning in changelog?

Reply:
Will add